common/rpctypes/types.go

```diff
@@ -147,3 +147,11 @@ type AddressesResponse struct {
 type PeersResponse struct {
 	Addrs []string `json:"addresses" validate:"dive,required"`
 }
+
+// PairsRequest ...
+type PairsRequest = DiscoverRequest
```

Comment:
since the `Provides` field of the request is always set to `""`, can you define this as a new type:
```
type PairsRequest struct {
    SearchTime uint64 `json:"searchTime"` // in seconds
}
```